sdk/webpubsub/azure-messaging-webpubsub/pom.xml

```diff
@@ -115,17 +111,17 @@
       <version>1.1</version>  <!-- {x-version-update;javax.websocket:javax.websocket-api;external_dependency} -->
       <scope>test</scope>
     </dependency>
-    <dependency>
-      <groupId>org.glassfish.tyrus.bundles</groupId>
-      <artifactId>tyrus-standalone-client</artifactId>
-      <version>1.16</version>  <!-- {x-version-update;org.glassfish.tyrus.bundles:tyrus-standalone-client;external_dependency} -->
-      <scope>test</scope>
-    </dependency>
     <dependency>
       <groupId>org.glassfish.tyrus</groupId>
-      <artifactId>tyrus-container-grizzly</artifactId>
-      <version>1.2.1</version>  <!-- {x-version-update;org.glassfish.tyrus:tyrus-container-grizzly;external_dependency} -->
+      <artifactId>tyrus-client</artifactId>
```

Comment:
Switched to the non-standalone client as the standalone client was unconditionally exporting `javax.websocket-api` which resulting in reading the module from two different sources, resulting in a compilation error.